pywinauto/application.py

```diff
@@ -588,29 +588,40 @@ def print_identifiers(ctrls, current_depth=1):
 
             indent = (current_depth - 1) * u"   | "
             for ctrl in ctrls:
-                output = indent + '\n'
-                output += indent + u"{class_name} - '{text}'    {rect}\t\n"\
+                if ctrl not in control_name_map.keys():
+                    continue
+                ctrl_text = ctrl.window_text()
+                if ctrl_text:
+                    # transform multi-line text to one liner
+                    ctrl_text = ctrl.window_text().replace('\n', r'\n').replace('\r', r'\r')

```

Comment:
Not sure, because I've done this for quite short string (say <30 chars) that was of 3 lines.